packages/react-dom/package.json

```diff
@@ -50,8 +50,10 @@
     "./profiling": "./profiling.js",
     "./test-utils": "./test-utils.js",
     "./unstable_testing": "./unstable_testing.js",
-    "./package.json": "./package.json",
-    "./": "./"
+    "./testing": "./testing.js",
```

Comment:
This is an experimental package. It got accidentally added to the `@next` release channel. We don't intend to release to 18 stable. I'll remove it in a follow-up.

Reply:
Will be fixed by @sebmarkbage in https://github.com/facebook/react/pull/23258